### PR TITLE
CSV demo with modifiers

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

**Comment by bre1470** [comment]
Looks good, @pawellysy ! 👍

File: samples/highcharts/data-tools/csv-modifiers/demo.js [hunk 1]
```diff
@@ -0,0 +1,271 @@
+const csvData = document.getElementById('csv').innerHTML;
+
+Highcharts.setOptions({
+
+    chart: {
+        type: 'line',
```
Comment (sebastianbochan) [inline samples/highcharts/data-tools/csv-modifiers/demo.js:6]: Line is the default type for HC, so you don't need to place it here.

File: samples/highcharts/data-tools/csv-modifiers/demo.js [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+        zooming: {
+            enabled: true,
+            type: 'x'
+        }
+    },
+    yAxis: {
+
```
Comment (sebastianbochan) [inline samples/highcharts/data-tools/csv-modifiers/demo.js:13]: Remove extra lines/spaces in the file.

**Review by sebastianbochan — CHANGES_REQUESTED** [review]

File: samples/highcharts/data-tools/csv-modifiers/demo.js [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+                    id: 'legend'
+                }]
+            }]
+        }]
+    },
+    components: [{
+        cell: 'asia-chart',
+        sync: {
+            visibility: true,
+            extremes: true
```
Comment (karolkolodziej) [inline samples/highcharts/data-tools/csv-modifiers/demo.js:97]: Do we need extremes sync here? It only causes issues with the `reset zoom` button.

**Review by karolkolodziej — APPROVED** [review]

**Review by sebastianbochan — APPROVED** [review]

File: samples/highcharts/data-tools/csv-modifiers/demo.js (hunk 3 above)
Comment (karolkolodziej) [reply samples/highcharts/data-tools/csv-modifiers/demo.js:97]: PS I reported this issue on Assana 😄

**Review by pawelfus — APPROVED** [review]
Thanks!

**Pull request merged**